aas_core_codegen/jsonld/main.py

```diff
@@ -254,7 +257,7 @@ def _generate_class_context(
 
     class_context_definition: JsonLdType = {}
     class_name = naming.json_model_type(cls.name)
-    uri_fragment = rdf_shacl_naming.class_name((Identifier(class_name)))
+    uri_fragment = rdf_shacl_naming.class_name((Identifier(cls.name)))
```

Comment:
```suggestion
    uri_fragment = rdf_shacl_naming.class_name(cls.name)
```

`cls.name` is already an `Identifier`.

Reply:
Thanks for the catch :) I added a new commit to remove the other unnecessary cast as well.